model/Export/AbstractQTIItemExporter.php

```diff
@@ -54,7 +54,7 @@
 
     public function buildAssetBasePath($path, $rename = true)
     {
-        return $this->buildBasePath() . $path;
+        return rtrim($this->buildBasePath(), '/') . '/' . $path;
```

Comment:
What do you think about:
rtrim($this->buildBasePath(), '/\\') . DIRECTORY_SEPARATOR . $path;